frontend/src/components/cell/cell-value.tsx

```
@@ -46,59 +47,71 @@ export const CellValue = observer(({ cell }: { cell: Cell }) => {
-          </Tooltip>
-        </div>
+            <Tooltip
+              text={<span><b>'{getCopiedText(capturedValue)}'</b> {t('widgets.labels.copy')}</span>}
+              placement="top"
+              trigger="click"
+            >
+              {cell.isEnum
+                ? <div className="deviceCell-text">{cell.getEnumName(cell.value)}</div>
+                : (
+                  <div>
+                    <span dangerouslySetInnerHTML={{ __html: formattedValue }}></span> {!!cell.units && (
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)